Vagrantfile

```diff
@@ -2,38 +2,45 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "ubuntu/trusty64"
+  config.vm.box = "ubuntu/xenial64"
 
   config.vm.provider "virtualbox" do |v|
     v.memory = 4096
     v.cpus = 2
   end
 
   config.vm.provision "shell", inline: <<-SHELL
+    # add docker repo
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu xenial stable"
+
+    # and golang 1.9 support
+    # add-apt-repository ppa:gophers/archive
+    add-apt-repository ppa:longsleep/golang-backports
```

Comment:
i'd prefer official installation, but this is fine too

Reply:
Reading the docs, I thought there wasn't a backport of 1.9 to xenial.
Just saw https://launchpad.net/~gophers/+archive/ubuntu/archive and it is there... Will change